envoy-control-tests/src/main/resources/envoy/bad_config.yaml

```diff
@@ -16,12 +16,6 @@ node:
   metadata:
     ads: true
     proxy_settings:
-      incoming:
-        endpoints:
-          - path: "/endpoint"
-            ## this is not a valid method
-            methods: [NOT_A_VALID_METHOD]
```

Comment:
This will no longer trigger an error. Envoy treats this as a string even though it is not a valid method.